server/lib/util.js

```diff
@@ -133,3 +135,32 @@ function isFalsy(value) {
     value.toLowerCase() === '0' ||
     value.toLowerCase() === '';
 }
+
+/**
+ * @function uniquelize
+ * @param {array} array An array in which we want to get only unique values
+ * @description return an array which contain only unique values
+ */
+function uniquelize (array) {
```

Comment:
I know you want the client/server code to look the same, but I think `lodash` has a functionality for this already...  [Here it is](https://lodash.com/docs/4.15.0#uniq).  If it does the same thing, it is probably better to use `lodash` in the long run.